activemodel/lib/active_model/errors.rb

```diff
@@ -329,19 +329,23 @@ def add(attribute, message = :invalid, options = {})
     #   person.errors.added? :name, "can't be blank" # => true
     #
     # If the error message requires an option, then it returns +true+ with
-    # the correct option, or +false+ with an incorrect or missing option.
+    # the correct option, or +false+ with an incorrect option.
     #
     #  person.errors.add :name, :too_long, { count: 25 }
+    #  person.errors.added? :name, :too_long                                # => true
     #  person.errors.added? :name, :too_long, count: 25                     # => true
     #  person.errors.added? :name, "is too long (maximum is 25 characters)" # => true
     #  person.errors.added? :name, :too_long, count: 24                     # => false
-    #  person.errors.added? :name, :too_long                                # => false
     #  person.errors.added? :name, "is too long"                            # => false
     def added?(attribute, message = :invalid, options = {})
       message = message.call if message.respond_to?(:call)
 
       if message.is_a? Symbol
-        details[attribute.to_sym].include? normalize_detail(message, options)
+        if options.blank?
```

Comment:
would something like this be simpler?
```ruby
normalized = normalize_detail(message, options)
details[attribute.to_sym].any? {|e| normalized.all? {|k,v| e[k] == v} } 
```

Reply:
Well, actually it isn't the same, see example

```ruby
person = Person.new
person.errors.add :name, :too_long, count: 25, actual_count: 42
# should return `false` since we are missing `actual_count: 42`,
# but with your implementation it returns `true`
assert_not person.errors.added? :name, :too_long, count: 2 
```